services/configuration/src/errors.rs

```diff
@@ -71,7 +73,10 @@ pub enum ErrorCode {
 // Common error types for `Propose` and `Vote`.
 #[derive(Debug, Fail)]
 pub(crate) enum Error {
-    #[fail(display = "Next configuration is already scheduled: {:?}", _0)]
+    #[fail(
+        display = "Next configuration is already scheduled: {:?}",
+        _0
+    )]
```

Comment:
This formatting is also sad. Meh. Whatever you say, rustfmt; whatever you say, mate...